stdlib/Profile/src/Profile.jl

```diff
@@ -63,6 +63,27 @@ macro profile(ex)
     end
 end
 
+export @profile_tasks
+
+"""
+    @profile_tasks
+
+`@profile_tasks <expression>` runs your expression while taking periodic backtraces of a sample of all live tasks (both running and not running).
+These are appended to an internal buffer of backtraces.
+
+It can be configured via `Profile.init`, same as the `Profile.@profile``, and that you can't use `@profile` simultaneously with `@profile_tasks`.
```

Comment:
```suggestion
It can be configured via `Profile.init`, same as the `Profile.@profile`, and that you can't use `@profile` simultaneously with `@profile_tasks`.
```